listenbrainz_spark/request_consumer/jobs/import_dump.py

```diff
@@ -32,16 +35,31 @@ def import_full_dump_by_id_handler(id: int):
     dump_name = import_dump_to_hdfs('full', overwrite=True, dump_id=id)
     return [{
         'type': 'import_full_dump',
-        'imported_dump': dump_name,
+        'imported_dump': [dump_name],
         'time': str(datetime.utcnow()),
     }]
 
 
 def import_newest_incremental_dump_handler():
-    dump_name = import_dump_to_hdfs('incremental', overwrite=False)
+    imported_dumps = []
+    latest_full_dump = utils.get_latest_full_dump()
+    if not latest_full_dump:
```

Comment:
Just so we're clear, full dump 1 + incremental dump 2 == full dump 2

So what I'm thinking we need to do for all imports is this:
* Figure out which ID the spark cluster is currently at (this doesn't depend on whether the last dump imported was full or incremental), say this ID is x
* If doing an incremental import, import all dumps that are available starting from x + 1
* If doing a full import, import the latest full dump that has value > x

Reply:
I have addressed issue number 1 and 2. Solving the 3rd one isn't easy as we don't know the latest dump available to download until we actually start downloading the dump. However this wont be an issue as all incremental dumps after the full dump will be imported on the next call to `import_newest_incremental_dump_handler`.

Reply:
You can always just list the dumps in the ftp dir and check. I think that should be possible, but I guess we can do it in a future PR.